go/libkb/passphrase_helper.go

```diff
@@ -40,6 +40,8 @@ func GetPaperKeyPassphrase(ui SecretUI, username string) (string, error) {
 	arg.Prompt = fmt.Sprintf("Please enter a paper backup key passphrase for %s", username)
 	arg.Features.StoreSecret.Allow = false
 	arg.Features.StoreSecret.Readonly = true
+	arg.Features.ShowTyping.Allow = true
+	arg.Features.ShowTyping.DefaultValue = true
```

Comment:
This is `getPaperKeyPassphrase()`, so we're only defaulting to show typing for paper keys, 'cause they're impossible to type blind.